.github/renovate.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "dependencyDashboard": false,
+  "enabledManagers": ["gradle"],
+  "ignorePaths": ["gradle/**", "solr/**", "buildSrc/**", "dev-docs/solr-missing-doclet/**"],
```

Comment:
We only care about versions recorded in `versions.props`, not inside every `*.gradle` file.

Reply:
Why ignore solr-missing-doclet specifically; I think this "ignorePaths" is too specific in this case.  In other words, why not ignore all of dev-docs?  Trying to reduce maintenance to this config line & to be simpler.

Also; no idea if it's possible to configure a GHA to be include-list instead of exclude-list.  Include-list is obviously much more sensible for this PR!

Reply:
Originally I allowed renovate to suggest upgrades to everything, including `dev-tools/scripts/requirements.txt`, webapp JS libs, Dockerfile (image versions), antora etc. Then I restricted it to only "gradle" (maven pkgs).

I think this will work to only consider the two files where we define versions:
```json
"includePaths": ["versions.props", "build.gradle"]
```